tests.ipynb

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
No notebooks in the lib. Move test to scripts that will be runnable with pytest or remove the notebook